app/controllers/collection_templates/build_controller.rb

```diff
@@ -24,7 +24,13 @@ def update
     when 'auto_clean'
       ImageEnhanceJob.new.perform(@collection_template)
     when 'image_clean'
+      collection = Collection.find(@collection_template.collection_id)
```

Comment:
I think you could probably shorten this to: `@collection_template.collection` no?